github/resource_github_organization_custom_properties.go

```diff
@@ -85,16 +85,19 @@ func resourceGithubCustomPropertiesCreate(d *schema.ResourceData, meta any) erro
 	for _, v := range allowedValues {
 		allowedValuesString = append(allowedValuesString, v.(string))
 	}
-	valuesEditableBy := d.Get("values_editable_by").(string)
 
 	customProperty := &github.CustomProperty{
-		PropertyName:     &propertyName,
-		ValueType:        valueType,
-		Required:         &required,
-		DefaultValue:     &defaultValue,
-		Description:      &description,
-		AllowedValues:    allowedValuesString,
-		ValuesEditableBy: &valuesEditableBy,
+		PropertyName:  &propertyName,
+		ValueType:     valueType,
+		Required:      &required,
+		DefaultValue:  &defaultValue,
+		Description:   &description,
+		AllowedValues: allowedValuesString,
+	}
+
+	if val, ok := d.GetOk("values_editable_by"); ok {
```

Comment:
question: Since this is a computed value, aren't there cases where `ok` would be false, but `val` has a value? Should we ignore the value in those cases?